Sdk/Speckle.Connectors.Utils/SpeckleLogProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.Logging;
+using Speckle.Connectors.Logging;
+
+namespace Speckle.Connectors.Utils;
+
+public sealed class SpeckleLogProvider(Logger speckleLogger) : ILoggerProvider
+{
+  public void Dispose() { }
+
+  public ILogger CreateLogger(string categoryName) => new SpeckleLogger(speckleLogger);
```

Comment:
Are we not using the categoryName for anything? if so... should we?

Reply:
By default, it uses the class name when using ILogger<T>